src/utils/comparer.ts

```diff
@@ -0,0 +1,30 @@
+import { Group, Person } from '../database'
+
+export const PersonArrayComparer = (a: Person[], b: Person[]) => {
+    if (a.length !== b.length) return false
+    return a.every((person, index) => {
+        const target = b[index]
```

Comment:
Do we need a position-to-position compare?

Reply:
don't know... You may remove some if you find it unnecessary.

Reply:
Is there something like a deepequals library for this stuff?

Reply:
https://github.com/tc39/proposal-record-tuple is nice to us